Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyPluginLogic.h

```diff
@@ -83,6 +83,8 @@ class Q_SLICER_MODULE_SUBJECTHIERARCHY_WIDGETS_EXPORT qSlicerSubjectHierarchyPlu
   void registerCorePlugins();
 
   /// Get all view context menu actions available
+  Q_INVOKABLE void setEditPropertiesEnabled(bool enable);
```

Comment:
If you have a setter function you need to add a getter function too. Also make it a Qt property.